api/src/event/entities/booking-event.entity.ts

```diff
@@ -21,7 +21,7 @@ export class BookingEventEntity extends EventEntity {
       field: this.field,
       previous: this.previous,
       updated: this.updated,
-      user: `${this.user?.firstName} ${this.user?.lastName}`,
+      // user: `${this.user?.firstName} ${this.user?.lastName}`,
```

Comment:
@michel-fresh  this is the issue if we remove saving first_name to user table. instead, we have to save user_name in event entity, then `user: this.user`